packages/safe-apps-sdk/package.json

```diff
@@ -15,7 +15,7 @@
   ],
   "scripts": {
     "test": "jest",
-    "format-dist": "sed -i 's/\"files\":/\"_files\":/' dist/package.json",
+    "format-dist": "sed -i '' 's/\"files\":/\"_files\":/' dist/package.json",
```

Comment:
Since we are all using `macOS` i decided to change that. for more context check safe-apps-SDK readme